CHANGELOG.md

```diff
@@ -400,6 +413,7 @@ been added to each release, please refer to the [blog](https://blog.gitea.io).
   * Update the provided gitea.service to mention socket activation (#8531)
   * Doc added how to setup email (#8520)
 * MISC
+  * Backport Locales [2020-01-14] (#9773)
```

Comment:
```suggestion
```

Reply:
This one I think we left because we don't usually backport locales. Is it significant enough to leave on the changelog?

Reply:
Ah yeah, I guess we also had this already in RC changelog. Ignore this nit. Thanks :)